src/EnergyPlus/Pumps.cc

```diff
@@ -1084,19 +1084,19 @@ namespace Pumps {
             if (PumpEquip(PumpNum).PumpType == Pump_VarSpeed || PumpEquip(PumpNum).PumpType == Pump_ConSpeed ||
                 PumpEquip(PumpNum).PumpType == Pump_Cond) {
 
-                SetupOutputVariable("Pump Electric Energy",
+                SetupOutputVariable("Pump Electricity Energy",
```

Comment:
I don't see Pump Electricity in the report variables csv.

Reply:
They are in B281 & B282.